### PR TITLE
[WIP] gui/user management: update lang strings. 

### PR DESCRIPTION
**Pull request by Croden1999**
# About
add some text lang strings, and also want to complete user management lang, would like to request devlopers to help with this pr, thx a lot.

- request to Mr. @Zangetsu38 .

# Result

### CODE REVIEW

**Comment by Zangetsu38** [comment]
yyyou need learn how using git, and can squarsh commit, and just edit some file without create 50 commit ^^

**Comment by Croden1999** [comment]
> you need learn how using git, and can squash commit, and just edit some file without create 50 commit ^^

i got it, thx again, Mr. Zangetsu.

File: lang/zh-t.xml [hunk 1]
```diff
@@ -155,9 +155,9 @@
 	</indicator>
 
 	<initial_setup>
-		<back>Back</back>
-		<completed_setup>You have now completed initial setup.
-Your Vita3K system is ready!</completed_setup>
+		<back>返回</back>
+		<completed_setup>已完成初期設定。
+請盡情享受Vita3K。</completed_setup>
```
Comment (Zangetsu38) [inline lang/zh-t.xml:160]: remove for chinese in initial setup, i have explain, is totaly normal ^^, maybe one days can works, but for now i can't.

Comment (Croden1999) [reply lang/zh-t.xml:160]: i got it.

**Comment by Zangetsu38** [comment]
please, i said squash, you have again created 2 commit 😭, using no website for edit code, learn git, for can rebase/squash/edit last commit ^^

File: vita3k/gui/src/user_management.cpp [hunk 2]
```diff
@@ -398,7 +398,7 @@ void draw_user_management(GuiState &gui, HostState &host) {
         ImGui::PopStyleVar();
     } else if (menu == "confirm") {
         ImGui::SetWindowFontScale(0.8f);
-        const std::string msg = "The following user has been created";
+        const std::string msg = lang["user_created"].c_str();
```
Comment (Zangetsu38) [inline vita3k/gui/src/user_management.cpp:401]: fail here :D, string no need c_str() us for char that

**Comment by Croden1999** [comment]
> please, i said squash, you have again created 2 commit 😭, using no website for edit code, learn git, for can rebase/squash/edit last commit ^^

my fault, thx Mr. Zangetsu.

**Comment by Croden1999** [comment]
i pull again.

**Pull request closed without merging**